modules/33acrossBidAdapter.js

```diff
@@ -267,10 +306,60 @@ function _buildBannerORTB(bidRequest) {
     ? _getViewability(element, utils.getWindowTop(), minSize)
     : NON_MEASURABLE;
 
+  const ext = contributeViewability(viewabilityAmount);
+
   return {
     format,
-    ext: contributeViewability(viewabilityAmount)
+    ext
+  }
+}
+
+// BUILD REQUESTS: VIDEO
+// eslint-disable-next-line no-unused-vars
+function _buildVideoORTB(bidRequest) {
+  const videoAdUnit = utils.deepAccess(bidRequest, 'mediaTypes.video', {});
+
+  const video = {}
+
+  const sizes = videoAdUnit.playerSize || [];
+  const {w, h} = _getSize(sizes);
+  video.w = w;
+  video.h = h;
+
+  // Obtain all ORTB params related video from Ad Unit
+  VIDEO_ORTB_PARAMS.forEach((param) => {
+    if (videoAdUnit.hasOwnProperty(param)) {
+      video[param] = videoAdUnit[param];
+    }
+  });
+
+  const product = _getProduct(bidRequest);
+
+  // Placement Inference Rules:
+  // - If no placement is defined then default to 2 (In Banner)
+  // - If product is instream (for instream context) then override placement to 1
+  video.placement = video.placement || 2;
+
+  if (product === PRODUCT.INSTREAM) {
+    video.startdelay = video.startdelay || 0;
+    video.placement = 1;
+  };
+
+  // bidfloors
+  if (typeof bidRequest.getFloor === 'function') {
```

Comment:
@terryc33x  sorry again, I just remembered to add Bidfloors as well. Please review.